auditlog_security/views/auditlog_view.xml

```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+     <record model="ir.ui.view" id="view_auditlog_log_tree">
+        <field name="name">auditlog.log.form</field>
+        <field name="model">auditlog.log</field>
+        <field name="inherit_id" ref="auditlog.view_auditlog_log_tree" />
+        <field name="arch" type="xml">
+            <xpath expr="//tree" position="attributes">
+                <attribute name="delete">false</attribute>
```

Comment:
permissions should take care of that, right?

Reply:
Agreed, though I suspect that that's there because no human should be allowed to delete anything from the ui